### DISCUSSION TITLE
Quickstart is a Slowstart

### DISCUSSION THREAD

**Discussion by rollulus** [testcontainers/testcontainers-go#2829] [General]
Hello. The [quickstart](https://golang.testcontainers.org/quickstart/) suggests to install the module using `go get github.com/testcontainers/testcontainers-go` which installs v0.33.0. But the code snippet in the quickstart uses `testcontainers.CleanupContainer` which is in `master` but not in v0.33.0.

So either the quickstart should use `Terminate`, or a release should be created that has `CleanupContainer`.

Didn't want to create an bug/issue because it is demanding too much irrelevant information.

**Comment by mdelapenya** [comment]
Thanks @rollulus for opening this discussion which such a funny title :) 

There is an old and open issue regarding our docs system: https://github.com/testcontainers/testcontainers-go/issues/511

Hope it helps in clarifying the situation, which has not changed since then.

Thanks!

**Comment by rollulus** [comment]
Thanks. So if I understand it correctly the docs and releases can get out of sync. Would it help to instruct people to `go get go get github.com/testcontainers/testcontainers-go@main` then in the quickstart to avoid hitting the same problem (and future out of syncs)?

**Reply by mdelapenya** [reply]
Yeah, that sounds like a good idea. Could you please send a fix for that if you have time? Contributions are always welcome, and I'm even happier to review docs ones, as I also consider them real bugs.

**Reply by rollulus** [reply]
Will do, thank you!

**Reply by rollulus** [reply]
https://github.com/testcontainers/testcontainers-go/pull/2830

**Comment by mdelapenya** [comment]
Given `v0.34.0` is out, the function mentioned in this discussion is already available as public API, so no more confusion about it.

I'm closing this discussion as solved, even though we acknowledge that #511 is still around.

Thanks for your time reporting it.

**Reply by rollulus** [reply]
Thank you!
